src/components/Dashboard/Notetaker/Category/CategoryElements.jsx

```diff
@@ -27,17 +27,17 @@ export const CategoriesSidebarHeaderTitle = styled.h3`
     font-family: "Roboto Mono", monospace;
 `;
 export const CategoriesListContainer = styled.ul`
-    flex: ${(props) => (props.required || props.addMode ? "" : 1)};
+    flex: ${(props) => (props.required || props.$addMode ? "" : 1)};
     margin-bottom: ${(props) => (props.required ? "5px" : "0")};
     display: flex;
     flex-direction: column;
     gap: 5px;
     border: 2px solid #111;
     border-top: 0;
-    border-bottom: ${(props) => (props.addMode ? "0px" : "2px solid #111111")};
+    border-bottom: ${(props) => (props.$addMode ? "0px" : "2px solid #111111")};
     padding: 5px;
     overflow-y: auto;
-    z-index: ${(props) => (props.addMode ? "-100" : 0)};
+    z-index: ${(props) => (props.$addMode ? "-100" : 0)};
 `;
```

Comment:
This is just my opinion (totaly not urgent) but you could destructor the props like we do in every other componenet (and well, this is a component) something like this :

```js
export const CategoriesListContainer = styled.ul`
    flex: ${({ required, $addMode }) => (required || $addMode ? "" : 1)};
    margin-bottom: ${({ required }) => (required ? "5px" : "0")};
    display: flex;
    flex-direction: column;
    gap: 5px;
    border: 2px solid #111;
    border-top: 0;
    border-bottom: ${({ $addMode }) => ($addMode ? "0px" : "2px solid #111111")};
    padding: 5px;
    overflow-y: auto;
    z-index: ${({ $addMode }) => ($addMode ? "-100" : 0)};
`;
```